source/registry/selectors/index.md

```diff
@@ -0,0 +1,160 @@
+
+It is not possible to select a point using URI Fragments with the Media Fragment specification, as zero-sized fragments are not allowed. In order to fulfill the use cases, this specification defines a new Selector class called `PointSelector`.  
+
+| Property | Description                                            |
+| -------- | -----------------------------------------------------  |
+| type     | Required.  Must be the value "PointSelector".          |
+| x        | Optional. An integer giving the x coordinate of the point, relative to the dimensions of the target resource.  |
+| y        | Optional. An integer giving the y coordinate of the point, relative to the dimensions of the target resource.    |
+| t        | Optional. A floating point number giving the time of the point in seconds, relative to the duration of the target resource. |
+{: .api-table}
+
+For example, to select a point in a video that is 10 pixels in from the top left hand corner of the visual content, and 14.5 seconds into the duration:
+
+{% include api/code_header.html %}
+``` json-doc
+{
+  "type": "PointSelector",
+  "x": 10,
+  "y": 10,
+  "t": 14.5
+}
+```
+
+### 2.3 Content Selectors
+{: #content-selectors}
+
+Video content resources consist of both visual and audio content within the same bit-level representation. There are situations when it is useful to refer to only one aspect of the content -- either the visual or the audio, but not both. For example, an Annotation might associate only the visual content of a video that has spoken English in the audio, and an audio file that has the translation of that content in Spanish.
+
+This specification defines two Selectors, `AudioContentSelector` that selects the audio content and a second `VisualContentSelector` for the visual content. Neither selector has any additional properties.
+
+{% include api/code_header.html %}
+``` json-doc
+{ "type": "AudioContentSelector" }
+```
+
+{% include api/code_header.html %}
+``` json-doc
+{ "type": "VisualContentSelector" }
+```
+
+## Appendices
+
+### A. Acknowledgements
+
+The production of this document was generously supported by a grant from the [Andrew W. Mellon Foundation][org-mellon].
+
+Thanks to the members of the [IIIF][iiif-community] for their continuous engagement, innovative ideas and feedback.
+
+### B. Change Log
+
+| Date       | Description                                        |
+| ---------- | -------------------------------------------------- |
+| 2023-04-27 | Version 1.2 (Change document to Registry of Selectors from Open/Web Annotation Extensions)
+| 2018-03-14 | Version 1.1 (Update ImageApi, add Point Selectors) |
+| 2014-07-01 | Version 1.0 (RFC)                                  |
+{:.api-table}
+
+[prezi-rot]: {{ site.api_url | absolute_url }}/presentation/2.0/#rotation "Rotation in Presentation API"
```

Comment:
I think \[prezi-rot\] should be in `links.md` and we should also reference prezi 3